### DISCUSSION TITLE
Blacklist - Don't display blocked posts at all

### DISCUSSION THREAD

**Discussion by SquiddyKy** [AprilSylph/XKit-Rewritten#656] [Suggestion Box]
Hi! First up, April, I just want to say thank you for the years and years of work you've put into both New Xkit and Xkit Rewritten (and tumblr itself). A fair part of the reason I'm able to easily use tumblr is because of you, so thanks.

Second: I know that you've made your stance clear that you are not planning on implementing any version of Blacklist, but I'd really appreciate it if you'd reconsider because of the 'Don't display blocked posts at all' function that tumblr's content filtering lacks.

While I am incredibly grateful for tumblr's content filtering, and I do use it a lot for posts that I'd like to be warned contain certain topics before interacting with them, because it currently just censors the post and says 'This contains X', it's really not useful for me in other matters. I have a number of triggers that are word-based, and (to use tumblr's example) having my dash interrupted with 'This post contains filtered tags: #horse' is still reminding me of horses, and can cause an anxiety attack.

Meanwhile, Xkit's Blacklist had the ability to banish any topics I didn't want to even be reminded of to the nether realm. I know it's had the 'not recommended' note next to it from the beginning, but honestly it's been more helpful than I can really express. (Other than triggers, it's also been pretty helpful when one of my mutuals starts spamming my dash with a fandom I don't particularly care about, and I'd like the ability to see posts from literally any other blog I'm following without dredging through pages of anime). It gives me a level of control over my dash that means I can actually relax while scrolling, instead of always being on edge.

I don't know if this is maybe a feature that tumblr staff are planning on implementing at any point in the future, and I understand if you just point-blank don't want to go through the hassle of adding a feature like this into Xkit Rewritten. I'd just really like to not have to have multiple add-ons installed to make tumblr usable.

Thanks for reading all this, and I hope you have a nice day

Edit: Whoops, forgot to mention the 'Hide filtered posts entirely' under tweaks, knew I was missing something XD
While I really really appreciate this option, and it gets close to what I want, I still like to have some sort of distinction between posts I need warning for (that I use tumblr's filtering system for) and posts containing subjects I want hidden entirely (which blacklist is doing currently). Again, I entirely understand if this is a level of granularity you're not interested in sorting, but I want to put the suggestion out there anyway.

**Comment by opensourcenebula** [comment]
Glad to see someone else came in with this idea beforehand. Sorry to bring up a month old post but I'd like to very loudly second the notion.

Right now all of my filtered tags are triggers where just the word is bad enough, so having the "completely remove filtered posts" option works for me _personally,_ but there's plenty of other users who want at least _some_ granularity. Right now its all or nothing. You can either block every filtered post from appearing and lose your ability to "mute" annoying (but non-triggering) posts, or be subjected to potentially triggering posts completely at random.

I'd love to see at least a little division, or maybe a separate "mute" option of some kind. I'm on the same page as OP that I understand why you wouldn't, but its an accessibility option that would benefit a lot of users. The current all-or-nothing that we have isn't unlivable but it makes the site just that little bit harder to use.

At the absolute minimum, I feel like something to reduce the appearance of blacklisted posts would be a major improvement over the current system of "HEY CHECK OUT THIS POST YOU BLOCKED THAT HAS THIS TRIGGER WORD IN IT!"
